### ISSUE TITLE
Homepage UI / UX Updates / Requests

### ISSUE THREAD

**Issue by jnthnvctr** [web3-storage/web3.storage#169]
1. Users who log-in should get landed on the Profile tab

I think we want to land folks on the "Profile/Account" page when they first log in. If our core audience is developers, feels like we want to plop them on the page that they can sign up for their token, hop in to the docs. I think our files page (although it works well for manual uploads) might derail folks into thinking the primary way to interact with the service is manual uploads vs. integrations (and maybe on that "Getting Started" section we might want a card for "Just give it a test" 

2. Copy edit changes on the front page to better land our message. 

"Build on Filecoin, no Infrastructure required" -> "Build apps backed by Filecoin, no infrastructure required."
(Paragraph under Decentralized Storage in 5 Minutes...) -> "Build applications with data persisted by Filecoin and available over IPFS. Get started in minutes with this simple interface."

3. Not sure if this is a bug -> a user tried creating a file (index.js) and literally copy and pasting the code on the front page and was unsuccessful (after an NPM install). Should we add a comment that they need to create a package.json (and link to the docs?)

4. On the design of the "Decentralized storage in 5 minutes" + Code snippet - suggstion from design to change the background color to #0b0a46 for better contrast

![image](https://user-images.githubusercontent.com/21000523/127028299-3b5c3acd-c2b5-4131-bb84-c60296c86086.png)

5. Update description under "Getting Started on Web3.Storage" to the following two lines:
"Storing data on open, public infrastructure shouldn't be a battle. 
Start building applications backed by Filecoin in 3 steps, it's that easy. "

![image](https://user-images.githubusercontent.com/21000523/127028582-a0fc13f8-6f33-4b18-b7ca-4a46d741658b.png)

**Comment by jnthnvctr** [comment]
- [ ] Users who log-in should get landed on the Profile tab
- [ ] Copy edit changes on the hero asset to better land our message.
- [ ] Investigate how to better make the code snippet useful for new devs
- [ ] Change background color of the "Decentralized Storage in... " + Code snippet section
- [ ] Copy edit to the paragraph under "Getting Started on Web3.Storage"

**Comment by rafaelramalho19** [comment]
We can consider adding a template project in github and add something like:
"If you don't know how to setup a project up, clone this start-up project in github"

**Comment by rafaelramalho19** [comment]
All other points other than number 3 are fixed in: https://github.com/web3-storage/web3.storage/pull/184

**Comment by jnthnvctr** [comment]
Re point number 3 - i believe this requires updating the copy to suggest that the file is saved as a .mjs file and run the file with node (via a commented out line) 

cc @alanshaw  who suggested that [here](https://protocollabs.slack.com/archives/C02750576C8/p1627324978317200?thread_ts=1627320274.311700&cid=C02750576C8).

**Comment by atopal** [comment]
We can also link to the docs from under the snippets, but really whatever we put there should be copy/pasteable

**Comment by jnthnvctr** [comment]
One more for the homepage - I think our [hero asset](https://www.figma.com/file/826Zurcs6kgWSzQ35it966?node-id=1396:565#96818657) is actually slightly further down the page than was intended in the Figma, could we adjust this up?

**Comment by jnthnvctr** [comment]
@atopal  maybe we change the "Learn More" to say "Read the Docs" as it does drop you directly into the docs if you click that link. Thoughts?

**Comment by jnthnvctr** [comment]
@rafaelramalho19  - I added the previously raised issue to the list above (this [PR](https://github.com/web3-storage/web3.storage/pull/158) is meant to address, but based on the preview I'm not sure it does) re: the "Decentralized storage in 5 minutes..." box expanding too far

**Comment by atopal** [comment]
"Read the docs" sounds good to me @jnthnvctr, I'd drop people into the getting started guide.

**Comment by jnthnvctr** [comment]
@rafaelramalho19 - I'm updating the checklist above based on some of the feedback on the Homepage that we got from our call with Andrew just now:

1. The hero asset (in Safari) seems to have a single line of white pixels on the left hand side. Also, when resizing the browser we ran into more white lines appearing

![image](https://user-images.githubusercontent.com/21000523/127187074-a3fb0185-c5ad-4dbb-8762-b0321426e948.png)

2. In the "Decentralized Storage in 5 minutes..." - can we change the wording of the link? Per the above I think we could go with "Follow the Quickstart Guide" since our link goes there. 

3. In the "Store" copy-pastable code block, there were a few additional issues we identified (outside of including a comment to indicate that someone should save the file as MJX, install web3-storage package, run using NPM). Could we change the token line to the following: 
` const token = process.env.WEB3_STORAGE_TOKEN ` ? Otherwise if copied and pasted (naively) you'll accidentally upload your token. 

4. Could we change the file line from "./" to "path/to/your/file"? Unclear where users will be running this, and don't want them accidentally doing something crazy.
